docs/url-examples.md

```diff
@@ -139,6 +139,195 @@ Additional information:
 * [How to import CSV data](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-csv-data)
 
 
+## /api/v1/labels
+
+**Get a list of label names**
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/labels'
+```
+
+</div>
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/labels'
+```
+
+</div>
+
```

Comment:
Missing links with additional information about `/api/v1/labels` similar to `/api/v1/label/<labelName>/values` below